django/dispatch/dispatcher.py

```diff
@@ -161,6 +164,36 @@ def send(self, sender, **named):
         terminating the dispatch loop. So it's possible that all receivers
         won't be called if an error is raised.
 
+        Arguments:
+
+            sender
+                The sender of the signal. Either a specific object or None.
+
+            named
+                Named arguments which will be passed to receivers.
+
+        Return a list of tuple pairs [(receiver, response), ... ].
+        """
+        if not self.receivers or self.sender_receivers_cache.get(sender) is NO_RECEIVERS:
+            return []
+        responses = []
+        for receiver in self._live_receivers(sender):
+            if asyncio.iscoroutinefunction(receiver):
+                response = async_to_sync(receiver)(signal=self, sender=sender, **named)
+            else:
+                response = receiver(signal=self, sender=sender, **named)
+            responses.append((receiver, response))
+        return responses
+
+    async def send_async(self, sender, **named):
```

Comment:
The cache uses an "a" prefix instead of `_async` suffix. Thoughts @andrewgodwin?